app/i18n/en/conf.php

```diff
@@ -48,7 +48,13 @@
 			'timeout' => 'HTML5 notification timeout',
 		),
 		'show_nav_buttons' => 'Show the navigation buttons',
-		'theme' => 'Theme',
+		'theme' => array(
+			'_' => 'Theme',
+			'deprecated' => array(
+				'_' => 'Deprecated',	// TODO
```

Comment:
Btw, this should also say "No longer supported" then :-) (Or is that what the // TODO is saying?)

Reply:
For me it is fine to keep it marked as "depricated", because the explanation is in the next line
![grafik](https://user-images.githubusercontent.com/1645099/211671462-70aa5ce5-1d71-43d6-95f1-64f8cb2f22be.png)